CMakeLists.txt

```diff
@@ -730,6 +730,7 @@ install(TARGETS ${MXNET_INSTALL_TARGETS}
 )
 
 install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+install(DIRECTORY 3rdparty/tvm/nnvm/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```

Comment:
Could you please add comments to where are we installing things by default? it's quite hidden in CMAKE documentation where they go. ( I had to dig a bit )

Reply:
That's a bit out of scope for what I intended with the PR, but sure.
You can find CMake's documentation [here](https://cmake.org/cmake/help/git-stage/command/install.html).

Reply:
See a005e9c24630bf0df0fa5f0324420ee6a5f19ada

Reply:
Thanks for that.